docker/centos/8/cpu_ispc_build/Dockerfile

```diff
@@ -1,3 +1,9 @@
+
```

Comment:
Empty first line here and many others is really not needed.

To my taste "Copyright" should be on the very first line, but one line with empty comment is ok as well.